doobie/src/io/github/iltotore/iron/doobie.scala

```diff
@@ -0,0 +1,27 @@
+package io.github.iltotore.iron
+
+import cats.Show
+import doobie.util.{Put, Get, Meta}
```

Comment:
You should use `_root_.doobie` instead to avoid cyclic reference. See other modules (like `cats.scala`) for example.